core/spec.md

```diff
@@ -436,6 +440,117 @@ attributes. However they MUST adhere to the following rules:
   example, use of a model (or domain) specific prefix could be used to help
   avoid possible future conflicts.
 
+#### `xid`
+
+##### Referencing xRegistry Entities
+
+An `xid` is a URI according to
+[RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986) that
+uniquely identifies an entity within an xRegistry. Its purpose is to
+reference the entity and express a relation to it, even if it is not stored in
+the local registry. The `xid` can also be used in other places where it
+is helpful to use a URI, e.g. events or knowledge graphs.
+Each entity has exactly one `xid`. Therefore, comparing two `xid`
+values is possible to determine, if they identify the same or different
+entities. To achieve this, an xRegistry-specific URI scheme `xreg` is defined.
+Below, [ABNF](https://datatracker.ietf.org/doc/html/rfc5234)  notation is used
+to describe how `xids` are structured:
+
+`xid` = `"xreg://" authority "/" entity-path`
+
+`entity-path` = `group-path` / `resource-path` / `version-path`
+
+`group-path`= `GROUPs "/" gID`
+
+`resource-path`= `group-path "/" RESOURCEs "/" rID`
+
+`version-path`= `resource-path "/" versions "/" vID`
+
+Where:
+
+- `GROUPs` is a Group type name (plural). e.g. `endpoints`.
+- `GROUP`, not shown, is the singular name of a Group type.
+- `gID` is the `id` of a single Group.
+- `RESOURCEs` is a Resource type name (plural). e.g. `definitions`.
+- `RESOURCE`, not shown, is the singular name of a Resource type.
+- `rID` is the `id` of a single Resource.
+- `vID` is the `id` of a single Version of a Resource.
+
+###### `authority`
+
+The `authority` is a string that uniquely identifies the entity's authority.
+It could be regarded as a namespace and SHOULD be based on a unique name
+like an entry in the domain name system (DNS) that is under the control of
+the entity's publisher.
+
+###### `entity-path`
+
+As xRegistry entities are organized in a hierarchy, the `entity-path`
+outlines the entity's position in the hierarchy. As the ABNF above shows,
+there can be paths for groups, resources, and versions.
+
+##### De-referencing a relative URI reference
+
+A relative URI reference can only be de-referenced if the base URI is known
+from the context.
+
+If the relative URI reference consists of a relative path segment, i.e. it
+does not begin with a slash, then the base URI is the URI of the surrounding
+entity.
+
+__Example:__
+
+```
+xid of surrounding entity: xreg://example.com/messagegroups/group2
+
+Reference: messages/message1
+
+De-referenced URI: xreg://example.com/messagegroups/group2/messages/message1
+```
+
+If the relative URI reference consists of an absolute path segment, i.e. it
+begins with a slash, then the base URI is the authority of the surrounding
+group.
```

Comment:
I think mentioning "group" here could be misleading, perhaps just "is the authority of the base URI" ?